testtools/httpserver_p.cpp

```diff
@@ -397,13 +397,15 @@ void HttpServerThread::run()
         }
 
         // TODO compared with expected SoapAction
```

Comment:
bonus points for implementing the TODO while at it ;)

Reply:
Done